src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,78 @@
+
+It's estimated that Eth2 will allow the network to process between 25,000 and [100,000 transactions per second](https://twitter.com/VitalikButerin/status/1312905884549300224?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1312905886327664640%7Ctwgr%5E%7Ctwcon%5Es2_&ref_url=https%3A%2F%2Fwww.coinspeaker.com%2Fvitalik-buterin-ethereum-layer-2%2F), with [100,000 as the theoretical maximum right now](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698).
+At the bare minimum, Eth2 will allow 64 times the amount of transactions as today which sits at around 15 transactions. That's the amount of shard chains (extra data and capacity) being introduced ([More on shard chains](/eth2/shard-chains/)). That means we can conservatively estimate how long it will take to process 100,000 transactions so we can compare it to the Visa example above.
+
+- `15 * 64 = 960` transactions per second.
+- `100,000 / 960 = 104.2` seconds to process 100,000 transactions.
+
+In 104.2 seconds, the Ethereum network will use the following amount of energy:
+
+`1.44kWh daily usage * 10,000 network nodes = 14,400kWh` per day.
+
+There are 86,400 seconds in a day, so `14,400 / 86,400 = 0.1666666667kWh` per second.
+
+If we times that by the amount of time it takes to process 100,000 transaction: `0.1666666667 * 104.2 = 17.3666666701 kWh`.
+
+That is **11.6554809866%** of the energy consumed by the same amount of transactions on Visa.
+
+And remember, this is based on the minimum amount of transactions that Eth2 will be able to handle per second. If Eth2 reaches its potential of 100,000 transactions per second, 100,000 transactions would consume 0.1666666667kWh.
+
+To put it another way, if Visa handled 140,839,000,000 transactions at a cost of 149 kWh per 100,000 transactions that's 209,850,110 kWh energy consumed for the year.
+
+Eth2 in a single year stands to consume 5,256,000 kWh. With a potential of 788,940,000,000 - 3,153,600,000,000 transactions processed in that time.
+
+_We’ve provided the basic comparison to Visa to baseline your understanding of Eth2 energy consumption against a familiar name. However, in practice, it’s not really correct to compare based on number of transactions. Ethereum’s energy output is time-based. If Ethereum did more or less transactions from one minute to the next, the energy output would stay the same._
+
+_It’s also important to remember that Ethereum does more than just financial transactions, it’s a platform for applications, so a fairer comparison might be to many companies/industries including Visa, AWS and more!_
+
+## Criticism
```

Comment:
> Criticism

I don't think the centralisation argument is all that relevant to a user who wants to know about how much energy Ethereum uses. I'd probably lean towards dropping this whole section.